Externals_CAM.cfg

```diff
@@ -10,7 +10,7 @@ local_path = src/dynamics/mpas/dycore
 protocol = git
 repo_url = https://github.com/MPAS-Dev/MPAS-Model.git
 sparse = ../.mpas_sparse_checkout
-tag = v8.0.1
+hash = ed6f8e39ec0a811b6d079ca0fc6f9fb6e30bad23
```

Comment:
Out of curiosity, was there a specific reason for updating the MPAS-A dycore to a newer hash?

Reply:
Thanks for the explanation! I agree that updating to the MPAS v8.1.0 release tag in a near-future PR sounds good.